mycroft/res/text/en-us/configurations.json

```diff
@@ -0,0 +1,33 @@
+{
+  "unit system": {
+    "metric": {"system_unit": "metric"},
+    "imperial": {"system_unit": "imperial"}
+  },
+  "location": {
+    "stockholm": {
+      "location": {
+        "city": {
+          "name": "Sockholm",
```

Comment:
Is this throwing shade at those Socks who live in Stockholm? :laughing:

Reply:
Lol, yeah! those darn socks, thinking they're better then the rest of us!


I'll fix...